packages/atlas/svgr.config.js

```diff
@@ -33,10 +34,11 @@ ${variables.imports}
 
 ${variables.interfaces}
 
-const ${variables.componentName} = (${variables.props}) => (
+const ${variables.componentName} = forwardRef((${variables.props}) => (
```

Comment:
Why do we need it?

Reply:
Check the code of `JoyTokenIcon`. To use the `reference` prop for the Tooltip I need to forward a reference to the icon.

Reply:
It was hard to find it 😛. Thanks!